README.md

```diff
@@ -1,93 +1,81 @@
-and call `bitrise run test`.
+<details>
+<summary>Description</summary>
 
-*Check the `bitrise.yml` file for required inputs which have to be
-added to your `.bitrise.secrets.yml` file!*
+Starts an Xcode simulator.
 
-Step by step:
+</details>
 
-1. Open up your Terminal / Command Line
-2. `git clone` the repository
-3. `cd` into the directory of the step (the one you just `git clone`d)
-5. Create a `.bitrise.secrets.yml` file in the same directory of `bitrise.yml`
-   (the `.bitrise.secrets.yml` is a git ignored file, you can store your secrets in it)
-6. Check the `bitrise.yml` file for any secret you should set in `.bitrise.secrets.yml`
-  * Best practice is to mark these options with something like `# define these in your .bitrise.secrets.yml`, in the `app:envs` section.
-7. Once you have all the required secret parameters in your `.bitrise.secrets.yml` you can just run this step with the [bitrise CLI](https://github.com/bitrise-io/bitrise): `bitrise run test`
+## 🧩 Get started
 
-An example `.bitrise.secrets.yml` file:
+Add this step directly to your workflow in the [Bitrise Workflow Editor](https://devcenter.bitrise.io/steps-and-workflows/steps-and-workflows-index/).
 
-```
-envs:
-- A_SECRET_PARAM_ONE: the value for secret one
-- A_SECRET_PARAM_TWO: the value for secret two
-```
-
-## How to create your own step
-
-1. Create a new git repository for your step (**don't fork** the *step template*, create a *new* repository)
-2. Copy the [step template](https://github.com/bitrise-steplib/step-template) files into your repository
-3. Fill the `step.sh` with your functionality
-4. Wire out your inputs to `step.yml` (`inputs` section)
-5. Fill out the other parts of the `step.yml` too
-6. Provide test values for the inputs in the `bitrise.yml`
-7. Run your step with `bitrise run test` - if it works, you're ready
+You can also run this step directly with [Bitrise CLI](https://github.com/bitrise-io/bitrise).
 
-__For Step development guidelines & best practices__ check this documentation: [https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md).
+### Examples
 
-**NOTE:**
+Boot simulator in the background and use it in the xcode-test Step:
+```yaml
+- xcode-start-simulator:
+    inputs:
+    - destination: platform=iOS Simulator,name=iPhone 8,OS=latest
+- xcode-test:
+    inputs:
+    - project_path: ./ios-sample/ios-sample.xcodeproj
+    - scheme: ios-sample
+    # Simulator
+    - destination: $BITRISE_XCODE_DESTINATION # Use the same destination as the xcode-start-simulator Step
+```
 
-If you want to use your step in your project's `bitrise.yml`:
+Detect if simulator timed out and restart the build:
+```yaml
+- xcode-start-simulator:
+    inputs:
+    - destination: platform=iOS Simulator,name=iPhone 8,OS=latest
+- trigger-bitrise-workflow:
+    is_always_run: true
+    run_if: '{{enveq "BITRISE_IS_SIMULAOR_TIMEOUT" "true"}}'
```

Comment:
Typo: `SIMULAOR`?

Looks like it's used consistently, so it may not be causing trouble.  But it might trip someone up in the future.